.licenserc.yaml

```diff
@@ -0,0 +1,12 @@
+header:
+  license:
+    spdx-id: MPL-2.0
+
+    content: |
+      This Source Code Form is subject to the terms of the Mozilla Public
+      License, v. 2.0. If a copy of the MPL was not distributed with this
+      file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
This step is necessary because the upstream workflow only knows the header text of the Apache license. I *could* eliminate this by sending them a PR, but haven't done so yet (they claim they welcome such things, so I suspect it would be well received, but four lines of YAML seems like the right call for this moment, rather than waiting on all of that)